BinaryEncoding.md

```diff
@@ -335,9 +332,9 @@ The `br_table` operator has an immediate operand which is encoded as follows:
 
 | Field | Type | Description |
 | ---- | ---- | ---- |
-| target_count | `uint16` | number of targets in the target_table |
-| target_table | `uint16*` | target entries that indicate an outer block or loop to which to break |
-| default_target | `uint16` | an outer block or loop to which to break in the default case |
+| target_count | `varuint32` | number of targets in the target_table |
+| target_table | `varuint32*` | target entries that indicate an outer block or loop to which to break |
```

Comment:
Wouldn't an interpreter interested in performance want to translate the table to a set of bytecode offsets anyway?

Also, uint16 gives us 65536 levels. There exist programs with switches with more arms than that. If this is to be a fixed size, I'd advocate for at least uint32_t.